### PR TITLE
CAS-65 refactor account activation endpoint

### PR DESCRIPTION
**Pull request by marcin-wiatrak**

### CODE REVIEW

**Comment by marcin-wiatrak** [comment]
test

**Review by dkkazmierczak — APPROVED** [review]

**Comment by marcin-wiatrak** [comment]
test2

**Review by marcin-wiatrak — COMMENTED** [review]
comment in review

File: src/app/api/auth/activate/route.ts [hunk 1]
```diff
@@ -1,93 +1,45 @@
-import { NextResponse } from 'next/server'
```
Comment (dkkazmierczak) [inline src/app/api/auth/activate/route.ts:1]: lipka

**Review by dkkazmierczak — APPROVED** [review]

**Pull request merged**